CSK_Module_MultiIOLinkSMI/scripts/CSK_MultiIOLinkSMI_Processing.lua

```diff
@@ -584,25 +706,33 @@ local function writeIODDMessage(messageName, jsonDataToWrite)
   end
   local errorMessage
   local messageWriteSuccess = true
-  if ioddWriteMessages[messageName].dataInfo.ProcessData and ioddWriteMessages[messageName].dataInfo.Parameters == nil then
-    dataToWrite = {ProcessData = dataToWrite}
-  elseif ioddWriteMessages[messageName].dataInfo.ProcessData == nil and ioddWriteMessages[messageName].dataInfo.Parameters then
-    dataToWrite = {Parameters = dataToWrite}
-  end
-  for dataMode, dataModeInfo in pairs(dataToWrite) do
-    for dataPointID, dataPointDataToWrite in pairs(dataModeInfo) do
-      local success = true
-      local errorCode
-      if dataMode == 'ProcessData' then
-        success, errorCode = writeProcessData(ioddWriteMessages[messageName].dataInfo.ProcessData[dataPointID], dataPointDataToWrite)
-      elseif dataMode == 'Parameters' then
-        success, errorCode = writeParameter(ioddWriteMessages[messageName].dataInfo.Parameters[dataPointID], dataPointDataToWrite)
+  if ioddWriteMessages[messageName] then
+    if ioddWriteMessages[messageName].dataInfo.ProcessData then
```

Comment:
```suggestion
    if ioddWriteMessages[messageName].dataInfo then
```
Otherwise it is not possible to write only ISDU (parameter) without process data

Reply:
merged